packages/ui/src/utils/UiHelperUtils.ts

```diff
@@ -22,19 +22,20 @@ export const UiHelperUtil = {
     return newUrl.hostname
   },
 
-  getTruncateString(string: string, chars: number, truncate: TruncateType) {
-    if (string.length <= chars) {
+  // eslint-disable-next-line max-params
```

Comment:
Remove `max-params` disable and change these to a single object argument i.e `getTruncateString({ string, charsStart ... })` We should do this in cases where there are 2+ func args to avoid mistakes of ordering them incorrectly, especially when they have same types like here `number`